### DISCUSSION TITLE
Installation failed with error: ''pipenv' is not recognized as an internal or external command"

### DISCUSSION THREAD

**Discussion by darshitac11** [Atri-Labs/atrilabs-engine#340] [Help: Installation & Start] [answered]
Command: `pipenv install atri`

Error: 

```
'pipenv' is not recognized as an internal or external command, operable program or batch file.
```

Author: Mahi Chowdary (Slack community)

**Comment by darshitac11** [comment, marked as answer]
1. Check if pipenv is installed in your system. Make sure you [satisfy the prerequisites](https://docs.atrilabs.com/getting-started/installation/install_prereqs).

2. Check if `pipenv` is added in the PATH environment variable.

3. Refer this [Stack overflow answer](https://stackoverflow.com/a/47676529).

**Comment by satyverse** [comment]
How to add the `pipenv` in the PATH , could anyone tell this
